cucumber/policy/features/cidr_restrictions.feature

```diff
@@ -37,3 +37,28 @@ from a particular network, defined by a CIDR in the policy
     """
        ["192.168.0.1/32", "192.168.1.10/32"]
     """
+
+  Scenario: Invalid CIDR restriction string
+
+    When I load a policy:
+    """
+    - !host
+      id: serviceA
+      restricted_to: an_invalid_cidr_string
+    """
+    Then there is an error
+    And the error code is "validation_failed"
+    And the error message is "policy_text Error at line 2, column 39 in  : Invalid IP address or CIDR range 'an_invalid_cidr_string'"
```

Comment:
Might be good to make this less specific so that it's something more like:

```
    And the error message includes "Invalid IP address or CIDR range 'an_invalid_cidr_string'"
```

Otherwise if you add a line to the policy for some reason you're going to have to go through and update all these error messages for no good reason.

Reply:
Good call!